pkg/testcase/coredns.go

```diff
@@ -1,40 +1,33 @@
 package testcase
 
 import (
-	"log"
-
 	"github.com/rancher/distros-test-framework/pkg/assert"
 	"github.com/rancher/distros-test-framework/shared"
 
-	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
-func TestCoredns(deployWorkload bool) {
-	if deployWorkload {
-		_, err := shared.ManageWorkload("create", "dnsutils.yaml", "arch")
-		Expect(err).NotTo(HaveOccurred(),
-			"dnsutils manifest not deployed", err)
-	}
+func TestCoredns(deleteWorkload bool) {
+	_, err := shared.ManageWorkload("apply", "dnsutils.yaml")
+	Expect(err).NotTo(HaveOccurred(),"dnsutils manifest not deployed")
 
-	_, err := shared.AddHelmRepo("traefik", "https://helm.traefik.io/traefik")
-	if err != nil {
-		log.Fatalf("failed to add Helm repo: %v", err)
-	}
+	_, err = shared.AddHelmRepo("traefik", "https://helm.traefik.io/traefik")
+	Expect(err).NotTo(HaveOccurred(), "failed to add Helm repo: %v", err)
 
 	kubeconfigFlag := " --kubeconfig=" + shared.KubeConfigFile
 	fullCmd := shared.JoinCommands("helm list --all-namespaces ", kubeconfigFlag)
 	err = assert.CheckComponentCmdHost(
 		fullCmd,
 		"rke2-coredns-1.19.402",
```

Comment:
Should this be hardcoded? @fmoral2

Reply:
definitely not lol. i can correct on my PR, seen that i have no idea what should be going here   :)